router/broker.go

```diff
@@ -88,6 +116,11 @@ func newBroker(logger stdlog.StdLog, strictURI, allowDisclose, debug bool, publi
 		debug:         debug,
 		filterFactory: publishFilter,
 	}
+	err := b.PreInitEventHistoryTopics(evntCfgs)
+	if err != nil {
+		logger.Printf("error parsing/initializing event history config %v", err)
+		panic(err)
```

Comment:
If `PreInitEventHistoryTopics` can normally fail then the error should be returned. It it can (or should) never fail then it should handle the panic internally. I think panic like that is somewhat of a code smell.

The error and log message are redundant, just panic with the message.